command/command_test.go

```diff
@@ -90,7 +89,7 @@ func newMockAPIServer(t *testing.T) (config.Config, map[string]func(*http.Reques
 		fmt.Fprint(w, string(respJSON))
 	}))
 
-	root, err := ioutil.TempDir("", "mackerel-agent-test")
+	root, err := os.MkdirTemp("", "mackerel-agent-test")
```

Comment:
I think it is more simple to use `testing.T.TempDir` that both `ioutil.TempDir` and following `os.RemoveAll`.